modules/oc-mirror-v2-about-dry-run.adoc

```diff
@@ -0,0 +1,9 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected-v2.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="oc-mirror-v2-about-dry-run_{context}"]
+= Verifying your selected images for mirroring
+
+You can use oc-mirror plugin v2 to perform a test run (dry run) that does not actually mirror any images. This allows you to review the list of images that would be mirrored. You can also use a dry run to catch any errors with your image set configuration early. When running a dry run on a mirror-to-disk workflow, the oc-mirror plugin v2 checks if all the images within the image set are available in its cache. Any missing images are listed in the `missing.txt` file. When a dry run is performed before mirroring, both `missing.txt` and `mapping.txt` contain the same list of images.
```

Comment:
"This allows you" is an: 

![Screenshot from 2024-06-21 12-18-18](https://github.com/openshift/openshift-docs/assets/57954076/adef7d5c-3584-4665-87d7-bf39e5b2bada)

Consider:

> This means that you can review...

Reply:
This enables you to review?

Reply:
"Enables" would also be considered anthrophmorphic. Best to state what the "user can do".